[2023-10-26T20:48:51.343+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:48:51.344+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T20:48:51.345+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:48:51.345+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:48:51.359+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:48:51.358+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T20:48:51.388+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:48:51.400+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.059 seconds
[2023-10-26T20:49:21.535+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:49:21.536+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T20:49:21.538+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:49:21.538+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:49:21.544+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:49:21.543+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T20:49:21.545+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:49:21.557+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.024 seconds
[2023-10-26T20:49:51.717+0000] {processor.py:157} INFO - Started process (PID=59) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:49:51.718+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T20:49:51.719+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:49:51.719+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:49:51.725+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:49:51.724+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T20:49:51.726+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:49:51.738+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T20:58:34.286+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:58:34.287+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T20:58:34.288+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:58:34.288+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:58:34.294+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:58:34.292+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T20:58:34.295+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:58:34.310+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.026 seconds
[2023-10-26T20:59:04.500+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:59:04.500+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T20:59:04.502+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:59:04.502+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:59:04.508+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:59:04.506+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T20:59:04.508+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:59:04.521+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T20:59:34.754+0000] {processor.py:157} INFO - Started process (PID=59) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:59:34.801+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T20:59:34.803+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:59:34.803+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:59:34.854+0000] {logging_mixin.py:151} INFO - [2023-10-26T20:59:34.809+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T20:59:34.893+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T20:59:34.928+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.178 seconds
[2023-10-26T21:00:04.987+0000] {processor.py:157} INFO - Started process (PID=68) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:00:04.988+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:00:04.989+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:00:04.989+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:00:04.996+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:00:04.994+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:00:04.996+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:00:05.009+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.024 seconds
[2023-10-26T21:00:35.223+0000] {processor.py:157} INFO - Started process (PID=77) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:00:35.224+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:00:35.227+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:00:35.227+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:00:35.235+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:00:35.233+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:00:35.235+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:00:35.246+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.026 seconds
[2023-10-26T21:01:05.382+0000] {processor.py:157} INFO - Started process (PID=86) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:01:05.383+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:01:05.385+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:01:05.384+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:01:05.391+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:01:05.389+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:01:05.392+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:01:05.406+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
[2023-10-26T21:01:35.568+0000] {processor.py:157} INFO - Started process (PID=95) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:01:35.568+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:01:35.570+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:01:35.569+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:01:35.574+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:01:35.573+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:01:35.601+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:01:35.613+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.047 seconds
[2023-10-26T21:03:35.919+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:03:35.920+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:03:35.921+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:03:35.921+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:03:35.928+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:03:35.926+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:03:35.929+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:03:35.943+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
[2023-10-26T21:04:06.076+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:04:06.077+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:04:06.079+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:04:06.079+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:04:06.087+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:04:06.086+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:04:06.089+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:04:06.115+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.042 seconds
[2023-10-26T21:04:36.237+0000] {processor.py:157} INFO - Started process (PID=59) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:04:36.237+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:04:36.239+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:04:36.239+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:04:36.244+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:04:36.243+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:04:36.244+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:04:36.257+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:05:06.397+0000] {processor.py:157} INFO - Started process (PID=68) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:05:06.398+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:05:06.399+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:05:06.399+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:05:06.405+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:05:06.403+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:05:06.405+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:05:06.416+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.021 seconds
[2023-10-26T21:05:36.572+0000] {processor.py:157} INFO - Started process (PID=77) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:05:36.573+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:05:36.574+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:05:36.574+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:05:36.580+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:05:36.579+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:05:36.581+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:05:36.592+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:06:06.742+0000] {processor.py:157} INFO - Started process (PID=86) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:06:06.743+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:06:06.745+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:06:06.745+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:06:06.750+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:06:06.749+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:06:06.751+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:06:06.762+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:11:16.872+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:11:16.873+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:11:16.874+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:11:16.874+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:11:16.880+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:11:16.878+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:11:16.880+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:11:16.893+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:20:52.416+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:20:52.417+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:20:52.419+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:20:52.419+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:20:52.424+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:20:52.423+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:20:52.425+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:20:52.438+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:21:22.576+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:21:22.576+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:21:22.578+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:21:22.577+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:21:22.583+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:21:22.581+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:21:22.583+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:21:22.595+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:21:52.768+0000] {processor.py:157} INFO - Started process (PID=59) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:21:52.770+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:21:52.771+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:21:52.771+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:21:52.777+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:21:52.776+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:21:52.778+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:21:52.790+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:22:22.942+0000] {processor.py:157} INFO - Started process (PID=68) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:22:22.943+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:22:22.944+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:22:22.944+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:22:22.949+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:22:22.948+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:22:22.950+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:22:22.962+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.021 seconds
[2023-10-26T21:22:53.103+0000] {processor.py:157} INFO - Started process (PID=77) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:22:53.104+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:22:53.105+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:22:53.105+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:22:53.110+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:22:53.109+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:22:53.110+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:22:53.122+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.021 seconds
[2023-10-26T21:23:23.280+0000] {processor.py:157} INFO - Started process (PID=86) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:23:23.281+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:23:23.284+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:23:23.283+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:23:23.288+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:23:23.287+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:23:23.289+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:23:23.300+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.021 seconds
[2023-10-26T21:23:53.468+0000] {processor.py:157} INFO - Started process (PID=95) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:23:53.468+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:23:53.470+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:23:53.470+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:23:53.475+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:23:53.474+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:23:53.476+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:23:53.488+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:24:23.651+0000] {processor.py:157} INFO - Started process (PID=104) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:24:23.652+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:24:23.653+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:24:23.653+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:24:23.658+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:24:23.657+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:24:23.659+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:24:23.670+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.020 seconds
[2023-10-26T21:24:53.811+0000] {processor.py:157} INFO - Started process (PID=113) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:24:53.812+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:24:53.814+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:24:53.814+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:24:53.821+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:24:53.820+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:24:53.822+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:24:53.834+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.024 seconds
[2023-10-26T21:25:24.005+0000] {processor.py:157} INFO - Started process (PID=122) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:25:24.006+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:25:24.007+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:25:24.007+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:25:24.012+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:25:24.011+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:25:24.013+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:25:24.041+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.038 seconds
[2023-10-26T21:25:54.220+0000] {processor.py:157} INFO - Started process (PID=131) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:25:54.220+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:25:54.222+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:25:54.222+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:25:54.227+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:25:54.226+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:25:54.227+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:25:54.239+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.021 seconds
[2023-10-26T21:26:24.414+0000] {processor.py:157} INFO - Started process (PID=140) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:26:24.415+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:26:24.417+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:26:24.417+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:26:24.424+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:26:24.422+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:26:24.425+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:26:24.438+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
[2023-10-26T21:26:54.570+0000] {processor.py:157} INFO - Started process (PID=149) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:26:54.571+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:26:54.573+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:26:54.573+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:26:54.579+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:26:54.577+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:26:54.579+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:26:54.592+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:27:24.753+0000] {processor.py:157} INFO - Started process (PID=158) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:27:24.754+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:27:24.755+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:27:24.755+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:27:24.761+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:27:24.759+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:27:24.761+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:27:24.774+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:27:54.907+0000] {processor.py:157} INFO - Started process (PID=167) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:27:54.908+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:27:54.909+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:27:54.909+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:27:54.914+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:27:54.913+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:27:54.914+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:27:54.928+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:28:25.051+0000] {processor.py:157} INFO - Started process (PID=176) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:28:25.052+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:28:25.053+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:28:25.053+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:28:25.059+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:28:25.057+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:28:25.059+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:28:25.071+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.021 seconds
[2023-10-26T21:28:55.229+0000] {processor.py:157} INFO - Started process (PID=185) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:28:55.230+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:28:55.232+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:28:55.232+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:28:55.238+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:28:55.236+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:28:55.238+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:28:55.250+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:29:25.413+0000] {processor.py:157} INFO - Started process (PID=194) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:29:25.414+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:29:25.415+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:29:25.415+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:29:25.422+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:29:25.421+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:29:25.422+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:29:25.436+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
[2023-10-26T21:29:55.561+0000] {processor.py:157} INFO - Started process (PID=203) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:29:55.562+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:29:55.564+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:29:55.564+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:29:55.569+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:29:55.568+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:29:55.570+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:29:55.583+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:30:25.732+0000] {processor.py:157} INFO - Started process (PID=212) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:30:25.733+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:30:25.735+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:30:25.735+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:30:25.741+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:30:25.739+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:30:25.742+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:30:25.754+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.024 seconds
[2023-10-26T21:30:55.888+0000] {processor.py:157} INFO - Started process (PID=221) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:30:55.888+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:30:55.890+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:30:55.890+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:30:55.897+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:30:55.896+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:30:55.898+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:30:55.909+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:31:26.077+0000] {processor.py:157} INFO - Started process (PID=230) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:31:26.078+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:31:26.079+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:31:26.079+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:31:26.085+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:31:26.083+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:31:26.085+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:31:26.097+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:31:56.236+0000] {processor.py:157} INFO - Started process (PID=239) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:31:56.237+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:31:56.238+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:31:56.238+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:31:56.243+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:31:56.242+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:31:56.244+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:31:56.259+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
[2023-10-26T21:32:26.410+0000] {processor.py:157} INFO - Started process (PID=248) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:32:26.410+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:32:26.412+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:32:26.412+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:32:26.418+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:32:26.417+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:32:26.419+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:32:26.430+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:32:56.577+0000] {processor.py:157} INFO - Started process (PID=257) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:32:56.579+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:32:56.581+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:32:56.581+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:32:56.587+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:32:56.586+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:32:56.588+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:32:56.599+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.026 seconds
[2023-10-26T21:33:26.735+0000] {processor.py:157} INFO - Started process (PID=266) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:33:26.736+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:33:26.737+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:33:26.737+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:33:26.744+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:33:26.742+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:33:26.744+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:33:26.757+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:33:56.901+0000] {processor.py:157} INFO - Started process (PID=275) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:33:56.902+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:33:56.904+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:33:56.904+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:33:56.911+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:33:56.909+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:33:56.912+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:33:56.924+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
[2023-10-26T21:34:27.068+0000] {processor.py:157} INFO - Started process (PID=284) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:34:27.069+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:34:27.070+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:34:27.070+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:34:27.076+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:34:27.074+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:34:27.076+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:34:27.088+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:34:57.209+0000] {processor.py:157} INFO - Started process (PID=293) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:34:57.210+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:34:57.212+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:34:57.211+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:34:57.218+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:34:57.217+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:34:57.218+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:34:57.233+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.026 seconds
[2023-10-26T21:35:27.362+0000] {processor.py:157} INFO - Started process (PID=302) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:35:27.363+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:35:27.365+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:35:27.365+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:35:27.371+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:35:27.369+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:35:27.371+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:35:27.383+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
[2023-10-26T21:35:57.548+0000] {processor.py:157} INFO - Started process (PID=311) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:35:57.548+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:35:57.550+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:35:57.550+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:35:57.555+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:35:57.554+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:35:57.556+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:35:57.568+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.021 seconds
[2023-10-26T21:36:27.674+0000] {processor.py:157} INFO - Started process (PID=320) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:36:27.675+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:36:27.677+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:36:27.676+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:36:27.683+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:36:27.682+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:36:27.684+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:36:27.697+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
[2023-10-26T21:36:57.858+0000] {processor.py:157} INFO - Started process (PID=329) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:36:57.858+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:36:57.860+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:36:57.860+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:36:57.866+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:36:57.865+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:36:57.867+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:36:57.879+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:37:28.031+0000] {processor.py:157} INFO - Started process (PID=338) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:37:28.032+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:37:28.033+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:37:28.033+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:37:28.038+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:37:28.037+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:37:28.039+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:37:28.050+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.021 seconds
[2023-10-26T21:37:58.214+0000] {processor.py:157} INFO - Started process (PID=347) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:37:58.215+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:37:58.216+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:37:58.216+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:37:58.223+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:37:58.221+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:37:58.223+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:37:58.236+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.024 seconds
[2023-10-26T21:38:28.398+0000] {processor.py:157} INFO - Started process (PID=356) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:38:28.399+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:38:28.400+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:38:28.400+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:38:28.405+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:38:28.404+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:38:28.406+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:38:28.418+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:38:58.553+0000] {processor.py:157} INFO - Started process (PID=365) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:38:58.553+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:38:58.555+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:38:58.555+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:38:58.560+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:38:58.559+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:38:58.561+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:38:58.574+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:39:28.735+0000] {processor.py:157} INFO - Started process (PID=374) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:39:28.736+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:39:28.738+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:39:28.738+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:39:28.745+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:39:28.743+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:39:28.745+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:39:28.762+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.029 seconds
[2023-10-26T21:39:58.895+0000] {processor.py:157} INFO - Started process (PID=383) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:39:58.896+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:39:58.898+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:39:58.897+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:39:58.903+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:39:58.902+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:39:58.904+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:39:58.917+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:40:29.082+0000] {processor.py:157} INFO - Started process (PID=392) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:40:29.083+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:40:29.084+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:40:29.084+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:40:29.090+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:40:29.088+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:40:29.090+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:40:29.104+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.023 seconds
[2023-10-26T21:48:52.836+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:48:52.837+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:48:52.839+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:48:52.839+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:48:52.854+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:48:52.845+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:48:52.891+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:48:52.937+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.103 seconds
[2023-10-26T21:49:23.062+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:49:23.062+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:49:23.064+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:49:23.064+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:49:23.071+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:49:23.070+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:49:23.073+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:49:23.085+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
[2023-10-26T21:49:53.247+0000] {processor.py:157} INFO - Started process (PID=59) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:49:53.249+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:49:53.253+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:49:53.253+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:49:53.258+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:49:53.257+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:49:53.259+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:49:53.271+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.026 seconds
[2023-10-26T21:50:23.418+0000] {processor.py:157} INFO - Started process (PID=68) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:50:23.419+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:50:23.420+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:50:23.420+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:50:23.425+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:50:23.424+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:50:23.425+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:50:23.438+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.022 seconds
[2023-10-26T21:50:53.576+0000] {processor.py:157} INFO - Started process (PID=77) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:50:53.577+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:50:53.578+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:50:53.578+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:50:53.586+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:50:53.584+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:50:53.587+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:50:53.603+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.029 seconds
[2023-10-26T21:51:23.729+0000] {processor.py:157} INFO - Started process (PID=86) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:51:23.730+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:51:23.731+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:51:23.731+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:51:23.737+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:51:23.736+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:51:23.738+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:51:23.749+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.021 seconds
[2023-10-26T21:52:22.796+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:22.797+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:52:22.799+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:22.799+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:22.805+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:22.803+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:52:22.805+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:22.818+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.024 seconds
[2023-10-26T21:52:24.835+0000] {processor.py:157} INFO - Started process (PID=42) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:24.836+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:52:24.838+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:24.838+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:24.854+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:24.851+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:52:24.855+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:24.870+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.038 seconds
[2023-10-26T21:52:49.950+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:49.951+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:52:49.952+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:49.952+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:49.964+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:49.963+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:52:49.965+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:49.979+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.030 seconds
[2023-10-26T21:52:54.017+0000] {processor.py:157} INFO - Started process (PID=52) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:54.019+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:52:54.020+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:54.020+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:54.031+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:54.030+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:52:54.032+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:54.046+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.030 seconds
[2023-10-26T21:52:58.042+0000] {processor.py:157} INFO - Started process (PID=53) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:58.042+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:52:58.044+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:58.044+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:58.056+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:52:58.054+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:52:58.056+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:52:58.069+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.029 seconds
[2023-10-26T21:53:01.102+0000] {processor.py:157} INFO - Started process (PID=54) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:53:01.103+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:53:01.105+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:53:01.104+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:53:01.116+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:53:01.115+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:53:01.117+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:53:01.129+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.028 seconds
[2023-10-26T21:53:31.264+0000] {processor.py:157} INFO - Started process (PID=63) to work on /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:53:31.265+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/Data_Pipelines.py for tasks to queue
[2023-10-26T21:53:31.267+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:53:31.267+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:53:31.274+0000] {logging_mixin.py:151} INFO - [2023-10-26T21:53:31.272+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/Data_Pipelines.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/Data_Pipelines.py", line 3, in <module>
    from airflow.providers.microsoft.mssql.operators.mssql import MsSqlOperator
ModuleNotFoundError: No module named 'airflow.providers.microsoft.mssql'
[2023-10-26T21:53:31.274+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/Data_Pipelines.py
[2023-10-26T21:53:31.287+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/Data_Pipelines.py took 0.025 seconds
